docs/DeveloperGuide.md

```diff
@@ -792,7 +802,7 @@ testers are expected to do more *exploratory* testing.
 ### Adding a guest
 
 Prerequisites: Only one guest to be added.
-      The `name` and `room` of the guest to be added should not exist in GuestBook.
+      The `name` or `room` of the guest to be added should not exist in GuestBook.
```

Comment:
Should this be `name` and `room`? Using "or" would mean that if the name does not exist in the GuestBook but the room does, a user can still add a new guest with a different name but same room?

Reply:
Agreed! Great logical catch!